chrome/run.js

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env node
```

Comment:
Can you add a license header here? 

See https://github.com/webcompat/webcompat-reporter-extensions/blob/master/firefox/addon.js#L1, for example: